### PR TITLE
Make sure Manage Disk Space doc is consistent on cache size properties

### PR DESCRIPTION
**Pull request by kbatuigas**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2414
Review deadline: 3 July 2024

## Page previews

[Manage Disk Space > Space management](https://deploy-preview-585--redpanda-docs-preview.netlify.app/current/manage/cluster-maintenance/disk-utilization/#space_management)
[Tiered Storage > Caching and chunking properties](https://deploy-preview-585--redpanda-docs-preview.netlify.app/current/manage/tiered-storage/#caching-and-chunking-properties)
[Tiered Storage > Configuration properties](https://deploy-preview-585--redpanda-docs-preview.netlify.app/current/manage/tiered-storage/#tiered-storage-configuration-properties)
[Object Storage Properties](https://deploy-preview-585--redpanda-docs-preview.netlify.app/current/reference/properties/object-storage-properties/)

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/pages/cluster-maintenance/disk-utilization.adoc [hunk 1]
```diff
@@ -237,7 +237,7 @@ Space management divides the disk space into different areas that can be managed
 * Reserved disk space (`disk_reservation_percent`) is the area of disk space that Redpanda does not use.
 ** As the disk space used by the other areas grows to their target sizes, the reserved space provides buffer space to avoid free disk space alerts.
 ** SSDs that run near capacity can experience performance degradation, so this buffer space prevents the disks from running at capacity.
-* Cache storage (the minimum of `cloud_storage_cache_size_percent` or `cloud_storage_cache_size`) is the maximum size of the xref:../tiered-storage.adoc#caching[disk cache] used by Tiered Storage. As the cache reaches its limit, new data added to the cache removes old data from the cache.
+* Cache storage (`cloud_storage_cache_size_percent`, or `cloud_storage_cache_size` which overrides the former) is the maximum size of the xref:../tiered-storage.adoc#caching[disk cache] used by Tiered Storage. As the cache reaches its limit, new data added to the cache removes old data from the cache.
```
Comment (JakeSCahill) [inline modules/manage/pages/cluster-maintenance/disk-utilization.adoc]:
```suggestion
* Cache storage (`cloud_storage_cache_size_percent` or `cloud_storage_cache_size`) is the maximum size of the xref:../tiered-storage.adoc#caching[disk cache] used by Tiered Storage.  As the cache reaches its limit, new data added to the cache removes old data from the cache. The `cloud_storage_cache_size` property takes precedence if both `cloud_storage_cache_size_percent` and `cloud_storage_cache_size` are set.
```

**Comment by kbatuigas** [comment]
@andrwng Does `cloud_storage_cache_size` always take precedence, or are there any instances where that behavior changes?

File: modules/manage/pages/cluster-maintenance/disk-utilization.adoc (hunk 1 above)
Comment (andrwng) [inline modules/manage/pages/cluster-maintenance/disk-utilization.adoc]: This isn't quite accurate: if both are set, Redpanda will honor the minimum of the two

**Review by andrwng — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "@antora/cli": "3.1.2",
     "@antora/site-generator": "3.1.2",
     "@asciidoctor/tabs": "^1.0.0-beta.5",
-    "@redpanda-data/docs-extensions-and-macros": "latest"
+    "@redpanda-data/docs-extensions-and-macros": "^3.5.5"
```
Comment (JakeSCahill) [inline package.json]: can we undo this change

File: modules/manage/partials/tiered-storage.adoc [hunk 3]
```diff
@@ -1127,28 +1127,30 @@ ifndef::env-kubernetes[Edit them with the `rpk cluster config edit` command.]
...
 
-| config_ref:cloud_storage_cache_chunk_size,true,tunable-properties[]
+| config_ref:cloud_storage_cache_chunk_size,true,properties/object-storage-properties[]
 | The size of a chunk downloaded into object storage cache. Default is 16 MiB.
 
-| config_ref:cloud_storage_cache_size_percent,true,cluster-properties[]
-| Maximum size (as a percentage) of the disk cache used by Tiered Storage. + \nThis property overrides `cloud_storage_cache_size`. Default is 10%.
+| config_ref:cloud_storage_cache_size_percent,true,properties/object-storage-properties[]
+| Maximum size (as a percentage) of the disk cache used by Tiered Storage. +
+If both this property and `cloud_storage_cache_size` are set, Redpanda honors the minimum of the two.
```
Comment (JakeSCahill) [inline modules/manage/partials/tiered-storage.adoc]:
```suggestion
If both this property and `cloud_storage_cache_size` are set, Redpanda uses the minimum of the two. 
```

File: modules/manage/partials/tiered-storage.adoc [hunk 4]
```diff
@@ -1651,64 +1653,64 @@ The following list contains cluster-level configuration properties for Tiered St
...
+| config_ref:cloud_storage_azure_shared_key,true,properties/object-storage-properties[]
 | Azure shared key.
 
 Required for ABS/ADLS.
 
-| config_ref:cloud_storage_cache_size_percent,true,cluster-properties[]
+| config_ref:cloud_storage_cache_size_percent,true,properties/object-storage-properties[]
 a| Maximum size (as a percentage) of the disk cache used by Tiered Storage.
 
 This property overrides `cloud_storage_cache_size`. Default is 10%.
```
Comment (JakeSCahill) [inline modules/manage/partials/tiered-storage.adoc]: Needs updating to:

> If both this property and `cloud_storage_cache_size` are set, Redpanda uses the minimum of the two.

File: modules/manage/partials/tiered-storage.adoc [hunk 5]
```diff
@@ -1651,64 +1653,64 @@ The following list contains cluster-level configuration properties for Tiered St
...
 
-| config_ref:cloud_storage_cache_size,true,cluster-properties[]
+| config_ref:cloud_storage_cache_size,true,properties/object-storage-properties[]
 | Maximum size of the disk cache used by Tiered Storage. Default is 20 GiB.
```
Comment (JakeSCahill) [inline modules/manage/partials/tiered-storage.adoc]: We should mention this here too:

> If both this property and `cloud_storage_cache_size_percent` are set, Redpanda uses the minimum of the two.

File: modules/reference/pages/properties/object-storage-properties.adoc [hunk 6]
```diff
@@ -573,12 +573,10 @@ Divide the object storage cache across the specified number of buckets. This onl
 
 === cloud_storage_cache_size_percent
 
-Maximum size of the cloud cache.
+Maximum size of the cloud cache as a percentage of unreserved disk space (see config_ref:disk_reservation_percent,true,cluster-properties[]). The default value for this option is tuned for a shared disk configuration. Consider increasing the value if using a dedicated cache disk.
 
 The property <<cloud_storage_cache_size,`cloud_storage_cache_size`>> controls the same limit expressed as a fixed number of bytes.
```
Comment (JakeSCahill) [inline modules/reference/pages/properties/object-storage-properties.adoc]: consider adding:

> If both this property and `cloud_storage_cache_size` are set, Redpanda uses the minimum of the two.

**Review by JakeSCahill — APPROVED** [review]
No blocking comments but would be good to add this information to other places in the docs:

> If both `cloud_storage_cache_size_percent` and `cloud_storage_cache_size` are set, Redpanda uses the minimum of the two.

**Review by JakeSCahill — APPROVED** [review]
